From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Thu, 18 Jun 2020 12:03:49 -0700
Subject: [PATCH] Remove unneeded import of boost header

This header was removed in boost 1.73.0. Note that upstream removed the
header in a commit combined with unrelated changes.

Upstream-Status: Inappropriate [upstream fixed in different commit]

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 src/import.cc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/import.cc b/src/import.cc
index eef13236..6f38d77a 100644
--- a/src/import.cc
+++ b/src/import.cc
@@ -50,7 +50,6 @@ namespace fs = boost::filesystem;
 #include <boost/assign/std/vector.hpp>
 using namespace boost::assign; // bring 'operator+=()' into scope
 
-#include <boost/detail/endian.hpp>
 #include <cstdint>
 
 extern PolySet * import_amf(std::string, const Location &loc);
